spring-integration-file/src/main/java/org/springframework/integration/file/remote/gateway/AbstractRemoteFileOutboundGateway.java

```diff
@@ -430,6 +432,20 @@ public void setFileExistsMode(FileExistsMode fileExistsMode) {
 		}
 	}
 
+	public void setChmodOctal(String chmod) {
```

Comment:
Spring string to int conversion doesn't honor that (0600) - it only applies to numeric literals.